xarray/core/computation.py

```diff
@@ -926,6 +926,86 @@ def earth_mover_distance(first_samples,
         return apply_array_ufunc(func, *args, dask=dask)
 
 
+def dot(*args, **kwargs):
+    """ dot(*arrays, dims=None)
```

Comment:
`dot(*arrays, *, dims=None)` is the way to write this with Python 3's keyword only arguments.

Reply:
Maybe we would keep this as `dot(*arrays, **kwargs)` as we did not yet drop python 2 support?

Reply:
I was confused. `def dot(*arrays, *, dims=None)` is not valid syntax in Python 3, either. (There can only be one single `*`)